site/_less/layouts/action-network-petitions/white-two-column.less

```diff
@@ -141,11 +145,106 @@
 
   .petition-info {
     @media screen and (min-width: @large-phone) {
-      padding: 0 1rem 0 0;
+      padding: 0 2rem 0 0;
     }
   }
 
+   [for="opt-in"] {
+     background-color: white;
+     color: @blue-800;
+     padding: 2rem 5px;
+  }
+
   .an-footer-cta {
     display: none;
   }
 }
+
+  .petition blockquote p{
```

Comment:
this should be written as 

```less
.petition {
  blockquote {
    p {
      // add code here
    }
  }
}
```

And combined with code below which would be nested beneath `.petition`